numalogic/connectors/druid/_druid.py

```diff
@@ -78,7 +82,7 @@ def build_params(
         "post_aggregations": post_aggregations or dict(),
         "filter": _filter,
         "dimensions": dimension_specs,
-        "context": {"timeout": TIMEOUT},
+        "context": {"timeout": TIMEOUT, "configIds": config_ids, "source": "numalogic"},
```

Comment:
Can we pass this in the config?

We can here concatenate two dicts together?.
Something like {**context_A, **context_B}

Reply:
We already have config_id there. Will make things easier.

Reply:
No, I think let's keep this for now. Ideally, we should pass the context in the function args. But this will do for now.